ui/app/utils/supported-login-methods.ts

```diff
@@ -17,7 +17,7 @@ export const BASE_LOGIN_METHODS = [
   },
   {
     type: 'userpass',
-    displayName: 'Username',
```

Comment:
lots of methods use username and password to login, changed this to match vault documentation so it's clear this is referencing the `userpass` auth type